pkg/rdt/config.go

```diff
@@ -893,7 +893,7 @@ func parseMBAllocation(raw []interface{}) (uint64, error) {
 	for _, v := range raw {
 		strVal, ok := v.(string)
 		if !ok {
-			return 0, fmt.Errorf("not a string value %q", v)
+			log.Warn("ignoring non-string (%T) MBA allocation %v", v, v)
```

Comment:
Hmm... are you sure a `continue` is not missing at the end of the `!ok` branch, after the warning ?

In the case of a failed type assertion (IOW `!ok`), I think always have `strVal == ""` after the test, so we practically continue to process the empty string. And that would end up again in the last fallback else-branch giving another warning...

Reply:
You're correct. It works but sort of incorrectly. Fixed now by adding `continue`